server/main_test.go

```diff
@@ -32,6 +34,10 @@ func TestMain(m *testing.M) {
 	testbridge.SetupForCommonTest()
 
 	RunInGoTest = true // flag for NewServer to known it is running in test environment
+	// Enable TopSQL for all test, and check the resource tag for each RPC request.
+	// This is used to detect which codes are not tracked by TopSQL.
+	topsqlstate.EnableTopSQL()
+	unistore.CheckResourceTagForTopSQLInGoTest = true
```

Comment:
@breeswish Now, all test case in `server` pkg will check the resource tag now.   

I try to enable this test check in other pkg, but will meet many error, so currently I only enable this test in `server` pkg.

Reply:
> but will meet many error

Does these error comes from we are not tagging the request correctly? If so, maybe we need to add tracking issues to track them and fix them.

Reply:
ok, I will create issue to track this. and some error are cause by test itself.